app/models/case_log.rb

```diff
@@ -447,7 +439,7 @@ def process_postcode_changes!
 
   def process_previous_postcode_changes!
     self.ppostcode_full = ppostcode_full.present? ? ppostcode_full.upcase.gsub(/\s+/, "") : ppostcode_full
-    process_postcode(ppostcode_full, "previous_postcode_known", "is_previous_la_inferred", "prevloc", "ppostc1", "ppostc2")
+    process_postcode(ppostcode_full, "previous_postcode_known", "is_previous_la_inferred", "prevloc", nil, nil)
```

Comment:
Do we need to pass `nil` here at all or can we remove it from the method signature if it's being removed everywhere?

Reply:
Part of the next batch of DB changes